pkg/cmd/grafana/apiserver/cmd.go

```diff
@@ -30,6 +36,11 @@ func newCommandStartExampleAPIServer(o *APIServerOptions, stopCh <-chan struct{}
 			devAcknowledgementNotice,
 		Example: "grafana apiserver --runtime-config=example.grafana.app/v0alpha1=true",
 		RunE: func(c *cobra.Command, args []string) error {
+			// Setup logging as the very first thing after we have read the desired verbosity level
+			if err := setupLogging(o.ExtraOptions.Verbosity); err != nil {
```

Comment:
With my other comments, I think we can setup the Grafana logger even earlier, in the https://github.com/grafana/grafana/blob/1208888bb618774228aefe65884e06316a06e714/pkg/cmd/grafana/apiserver/cmd.go#L80

Reply:
Kept the setup in the command for now